src/components/business-lookup/BusinessLookup.vue

```diff
@@ -10,7 +10,7 @@
         class="mt-5"
         autocomplete="chrome-off"
         menu-props="{ maxHeight: 380 }"
-        label="Business or Corporation Name or Incorporation Number"
+        label="Business Name or Incorporation/Registration Number"
```

Comment:
As this component is used in different UIs, is this change OK for those other UIs?

Or, would it better to pass in this label as a prop? (You could default the prop to the old label string so that the other UIs aren't affected.)

Reply:
Looks good.